src/runtime/runtime_stm32wle5.go

```diff
@@ -4,21 +4,125 @@
 package runtime
 
 import (
+	"device/stm32"
 	"machine"
 )
 
 type arrtype = uint32
 
 func init() {
-	// Currently the clock is not configured, which means
-	// the MCU runs at default reset clock speed (4Mhz).
-	// Code to initialize RCC and PLL can go here.
+
+	// Configure 48Mhz clock
+	initCLK()
 
 	// UART init
 	machine.Serial.Configure(machine.UARTConfig{})
 
 	// Timers init
 	initTickTimer(&machine.TIM1)
+
+	// Radio init
+	subGhzInit()
+}
+
+func initCLK() {
+
+	// Set Power Voltage Regulator Range 2
+	//stm32.PWR.CR1.ReplaceBits(0b10, stm32.PWR_CR1_VOS_Msk, stm32.PWR_CR1_VOS_Pos)
```

Comment:
Should either un-comment, or completely remove

Reply:
Removed